### DISCUSSION TITLE
Setting dark/light mode icons for extension

### DISCUSSION THREAD

**Discussion by ryanlua** [microsoft/MicrosoftEdge-Extensions#117] [Q&A] [answered]
I was wondering what I would use to be able to detect the browser's theme for my extension. 

![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/80087248/8eba0a2f-d6c4-4f78-8982-e6e24e48c050)
![image](https://github.com/microsoft/MicrosoftEdge-Extensions/assets/80087248/68109328-e078-4deb-a8af-2fd807aa9939)

I would like this so I could properly adjust my extension's icon to match the browser theme. Right now, it's barely visible to see the icon so any API or way to specify the icon would be great.

**Comment by Bhuvana-Priya** [comment, marked as answer]
Hi @RyanLua ,

Thank you for letting us know about this. We will take this as feedback and will let you know once we have any update.
